app/src/main/java/org/wikipedia/login/LoginActivity.java

```diff
@@ -180,6 +181,7 @@ private void onLoginSuccess() {
         Prefs.setReadingListPagesDeletedIds(Collections.emptySet());
         Prefs.setReadingListsDeletedIds(Collections.emptySet());
         ReadingListSyncAdapter.manualSyncWithForce();
+        WikipediaApp.getInstance().getBus().post(new LoginLogoutEvent());
```

Comment:
Same question. Maybe you can check my implementation in #632 to see if it works for you.